generator/internal/codegen_utils.h

```diff
@@ -63,6 +65,21 @@ std::string ServiceNameToFilePath(absl::string_view service_name);
  */
 std::string ProtoNameToCppName(absl::string_view proto_name);
 
+enum class NamespaceType { NORMAL, INTERNAL };
```

Comment:
I think these should be named `kNormal` and `kInternal`. After https://github.com/googleapis/google-cloud-cpp/pull/4844 lands, I think clang-tidy will complain about this.